### DISCUSSION TITLE
dna icon in react

### DISCUSSION THREAD

**Discussion by enyelsequeira** [lucide-icons/lucide#900] [Q&A] [answered]
Hello all, I have a question, I assume not all icons are in each package, for example I am trying to use this one in react, but it is not there? 
![image](https://user-images.githubusercontent.com/50872387/207934899-0c07ebf0-e917-4927-8a94-8ff9df10dc4d.png)

**Comment by karsa-mistmere** [comment, marked as answer]
Hi @enyelsequeira,

No, all icons should be present in every package, although this is a pretty recent addition (v0.97.0 to be exact), can you check which version of `lucide-react` you're using?

**Reply by enyelsequeira** [reply]
Hey @karsa-mistmere this, was the issue, I upgraded and it was fine sorry. I was using     "lucide-react": "^0.92.0",
